tests/conftest.py

```diff
@@ -193,7 +196,8 @@ def ml_server(
         targets=targets,
         model_location=trained_model_directory,
     ):
-        yield
+        with patch.object(sensor_tag, "_asset_from_tag_name", return_value="default"):
```

Comment:
Same as above. I would maybe expect the default behaviour to be setting the asset to "default" if not defined, which makes me thing the test project tag_list sets assets, and we for some reason want them to be assigned a default asset instead for this test.

Reply:
I think we wanted to explicitly raise an error during an actual workflow, which is why we don't have any default asset for a tag name. But agree with making comments here.